src/graphics/GraphvizWiringDiagrams.jl

```diff
@@ -219,7 +226,7 @@ function box_html_label(nin::Int, nout::Int, text_label::String;
     Graphviz.Html("""
       <TABLE BORDER="0" CELLPADDING="0" CELLSPACING="0">
       <TR><TD>$(orientation == TopToBottom ? input_label : output_label)</TD></TR>
-      <TR><TD $(html_attributes(attrs))>$(escape_html(text_label))</TD></TR>
+      <TR><TD $(html_attributes(attrs))>$((text_label))</TD></TR>
```

Comment:
This change shouldn't be in this PR, as it addresses a separate issue and also I suspect it needs to be handled more carefully.

Reply:
sorry - that hack wasn't supposed to leak into the PR!